test/functional/Valhalla/playlist.xml

```diff
@@ -24,6 +24,9 @@
 <playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../TestConfig/playlist.xsd">
 	<test>
 		<testCaseName>ValueTypeTests</testCaseName>
+		<variations>
+                  <variation>-XX:ValueTypeFlatteningThreshold=1</variation>
```

Comment:
you need another variation for the default case, `<variation>NoOptions</variation>`